roles/ci_network/tasks/apply-dnsmasq.yml

```diff
@@ -36,46 +36,73 @@
   become: true
   when: >
     ("'127.0.0.1' not in ansible_dns.nameservers") or
-    ("'forwarders' in cifmw_network_local_dns")
+    ("'forwarders' in cifmw_network_dnsmasq_config")
```

Comment:
care to mention that parameter in the defaults/main.yml? Even as a comment - that would be nice for ppl looking "only" at that file.

Reply:
Done.